packages/grafana-ui/src/components/NodeGraph/ViewControls.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { Button } from '../Button';
+import { stylesFactory, useTheme } from '../../themes';
+import { GrafanaTheme } from '@grafana/data';
+import { css } from 'emotion';
+
+const getStyles = stylesFactory((theme: GrafanaTheme) => ({
+  minusButton: css`
+    margin-left: 2px;
+  `,
```

Comment:
maybe use spacing value from the theme (it's 2px)
```suggestion
  minusButton: css`
    margin-left: ${theme.spacing.xxs};
  `,
```